config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/FlightPotion-4LY2S8QsSw-JdMGrF8FUsQ==.json

```diff
@@ -0,0 +1,124 @@
+{
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": 0,
+      "questIDLow:4": 1598
+    }
+  },
+  "properties:10": {
+    "betterquesting:10": {
+      "autoClaim:1": 0,
+      "desc:8": "Tired of jetpacks? Want normal flight with no strings attached? Look no further!\n\nUsing your brand new MV Assembler and a Chicken Trophy, you can craft a Potion of Flight, giving you 7 minutes of Creative flight with 8 uses before you have to make a new one.\n\n[note]Chicken Trophies are a rare drop from killing chickens.[/note]",
+      "globalShare:1": 0,
+      "icon:10": {
+        "Count:3": 1,
+        "Damage:2": 0,
+        "OreDict:8": "",
+        "id:8": "AWWayofTime:alchemyFlask"
+      },
+      "isMain:1": 0,
+      "isSilent:1": 0,
+      "lockedProgress:1": 0,
+      "name:8": "Flight Potion",
```

Comment:
There's already a quest for flight potions in the Blood Magic tab. How about renaming this one to make it clear this is just a preview of their real power?